plugins/woocommerce-blocks/assets/js/data/payment/default-state.ts

```diff
@@ -14,6 +14,79 @@ import {
 import { SavedPaymentMethod } from './types';
 import { STATUS as PAYMENT_STATUS } from './constants';
 
+function getDefaultPaymentMethod() {
```

Comment:
This now comes from the server. It can either be a string in the case of a "normal" payment method, or an object when it's a saved payment method